app/create.py

```diff
@@ -58,11 +58,13 @@ def post(self):
         else:
             if not self.params.given_name:
                 return self.error(400, _('Name is required.  Please go back and try again.'))
-        if not self.params.author_name:
-            if self.params.clone:
-                return self.error(400, _('The Original author\'s name is required.  Please go back and try again.'))
-            else:
-                return self.error(400, _('Your name is required in the "Source" section.  Please go back and try again.'))
+        
+        if not self.params.own_info:
```

Comment:
I think this will only check whether the own_info was provided. You should check whether it's set to yes or no